AppDB/test/unit/test_datastore_server.py

```diff
@@ -825,12 +820,210 @@ def test_zigzag_merge_join(self):
     flexmock(query).should_receive("limit").and_return(1)
     self.assertEquals(dd.zigzag_merge_join(query, filter_info, []), None)
 
-  def test_get_meta_data_key(self):
+  def test_index_deletions(self):
+    app_id = 'guestbook'
```

Comment:
Can we have this test Greeting and its metadata global since we're using it in multiple tests? Maybe a static object up at the top?